docs/source/specification.md

```diff
@@ -63,18 +63,102 @@ Together, these provide a framework that {ref}`TRE operator <infrastructure_role
 
 (satre_principles)=
 
-## Architectural Principles
+### Architectural Principles
 
-The SATRE specification has been developed based on the following principles:
-
-- TREs should be as easy as possible for end-users to use (_e.g._ {ref}`data consumers <project_roles>`) whilst still remaining secure.
-- TREs should take accessibility for all users, including those with disabilities, into account.
-- TRE deployments should be offered that support data of different levels of sensitivity (_e.g._ through a tiered system of technical controls and policies).
-- TREs conforming to the specification should be interoperable and provide a familiar end-user experience.
-- The specification will be managed and updated following an open, community-driven process, and will not be tied to a single vendor or implementation.
+Architectural principles influence and shape the way you design and deliver a SATRE-aligned TRE.
```

Comment:
I'm not sure this is right, aren't these architectural principles more like what we had in mind when writing the spec, rather than what TRE developers/operators need to consider? That's why they were supposed to be brief originally.

Reply:
@harisood I think we should decide what the purpose of this section is and who is supposed to read it - should a TRE operator looking to improve their TRE or a TRE developer starting from scratch read these principles before going to the spec? Some of the statements read like they should be spec statements, especially "Observability" - are we duplicating content here?

Reply:
I think I agree- the "Implications" of these principles _are_ the statements in the rest of the spec, so I think we just need the statement and rationale here.

Does this need to be formal (heading, statement, rationale), or can we reformat this as:

```
#### Usability: A TRE instance that works for all users minimises barriers to use, and provides a productive and accessible analysis environment for research.

There is often a trade-off between increased operational security and the usability of a TRE.
In order to maintain productivity, a TRE must balance these two competing aims.
The design and configuration of a TRE should allow all individuals involved with a TRE to effectively fulfil their roles.
```

Reply:
> architectural principles more like what we had in mind when writing the spec, rather than what TRE developers/operators need to consider?

No these are for TRE people to consider. They're not hard and fast requirements.

> the "Implications" of these principles are the statements in the rest of the spec

Not always the case, for instance a lot of the usability ones are legit implcations. Agree for any should statements though - maybe these should be moved into the spec?